src/main.cpp

```diff
@@ -61,6 +68,14 @@ int main(int argc, char* argv[]) {
   //basicFillRect = runner.getRenderer()->getBasicFillRect(NovelRT::GeoVector<float>(200, 200), NovelRT::RGBAConfig(0, 255, 255, 255), rectArgs);
   auto textRect = runner.getRenderer()->getTextRect(NovelRT::RGBAConfig(0, 255, 0, 255), 70, "Gayathri-Regular.ttf", rectArgs);
   textRect->setText("RubyGnomer");
+  auto rectArgs2 = NovelRT::NovelCommonArgs();
+  rectArgs2.startingPosition = novelChanArgs.startingPosition;
+  rectArgs2.startingPosition.setX(rectArgs.startingPosition.getX() - 800);
+  rectArgs2.layer = 0;
+  rectArgs2.orderInLayer = 1;
+  rectArgs2.startingRotation = 0.0f;
+
+  basicFillRect = runner.getRenderer()->getBasicFillRect(NovelRT::GeoVector<float>(200, 200), NovelRT::RGBAConfig(0, 255, 255, 255), rectArgs2);
```

Comment:
nit: Maybe give this a better name, like `playAudioButton`.

A small text overlay that says `Play Audio` might also be a nice touch :P